lib/fast_inserter/fast_inserter_base.rb

```diff
@@ -55,7 +57,7 @@ def initialize(params)
       # out the load. The second 'false' param means don't fill in the last group with nil elements.
       all_values = params[:values]
       all_values.uniq! if @options[:unique]
-      group_size = Integer(params[:group_size] || 10_000)
+      group_size = Integer(params[:group_size] || DEFAULT_GROUP_SIZE)
```

Comment:
`params[:group_size] || ENV['FAST_INSERTER_GROUP_SIZE'].to_i || DEFAULT_GROUP_SIZE` instead? thay way we don't have to upgrade the gem to change the default